app/questionnaire/router.py

```diff
@@ -136,7 +158,7 @@ def get_previous_location_url(self, location, routing_path):
         if block_id_index != 0:
             previous_block_id = routing_path[block_id_index - 1]
             previous_block = self._schema.get_block(previous_block_id)
-            if previous_block["type"] == "RelationshipCollector":
+            if previous_block["type"] == "RelationshipCollector":  # type: ignore
```

Comment:
This doesn't need to be ignored, I guess it is complaining you could get a `None` if so it should be:
```py
if previous_block and previous_block["type"] == "RelationshipCollector":
```

Reply:
Added, thanks!